integration/tests/cook/test_basic.py

```diff
@@ -1403,7 +1404,10 @@ def get_docker_info():
             self.assertTrue('9090/udp' in ports)
             self.assertEqual(instance['ports'][1], int(ports['9090/udp'][0]['HostPort']))
         finally:
+            job = util.load_job(self.cook_url, job_uuid)
+            self.logger.info(f'Job status is {job["status"]}: {job}')
             util.session.delete('%s/rawscheduler?job=%s' % (self.cook_url, job_uuid))
+            mesos.dump_sandbox_files(util.session, instance, job)
```

Comment:
This call might fail, e.g., on a Mac+Docker combo as the host IPs are not accessible. We should defend against that.

Reply:
Added `try`\`except` inside `dump_sandbox_files`